static/js/components/tiles/scatter_tile.tsx

```diff
@@ -214,6 +206,42 @@ export function ScatterTile(props: ScatterTilePropType): JSX.Element {
   }
 }
 
+/**
+ * Returns callback for fetching chart CSV data
+ * @param props Chart properties
+ * @returns Async function for fetching chart CSV
+ */
+function getDataCsvCallback(
+  props: ScatterTilePropType,
+  scatterChartData: ScatterChartData
+): () => Promise<string> {
+  return () => {
+    // Assume both variables will have the same date
```

Comment:
the fetchData function does not assume both variables will have the same date and we wouldn't want csv data to be different from the chart data? I did see your thread with Julia about this, but technically a user can put anything in the date field when generating the tile, so lets at least add a TODO to flag the difference. Maybe make one function for getting date and use it both here and the chart data fetch and add a TODO in the function to handle multiple dates. 

Another problem is that like map_tile, there will be some stat vars that get a special date set, so that also needs to be handled: https://github.com/datacommonsorg/website/blob/master/static/js/tools/scatter/util.ts#L72-L76

Reply:
Updated scatter_tile with a TODO to handle dates on a per-variable basis, and added capped stat var date handling